src/utils/perfherder/chartJs/perfherderFormatter.js

```diff
@@ -17,28 +17,27 @@ const generateInitialOptions = series => {
     tooltips: {
       callbacks: {
         footer: (tooltipItems, data) => {
-          const tooltipData = [];
-          let delta = 'n/a';
-          let deltaPercentage = 'n/a';
-          let dataset = 'n/a';
-          let currentData = 'n/a';
+          // get data from all points of selected series
+          const dataset = data.datasets[tooltipItems[0].datasetIndex].data;
+          // get data from selected point
+          const currentData = dataset[tooltipItems[0].index].y;
+          const tooltipData = []; // footer's text lines will be stored here
```

Comment:
make this the first line of the block, it is the complement to the line at the end of the block (`return tooltipData`)

Reply:
oooh this one, I love it, new for me :grin: